Makefile

```diff
@@ -60,19 +59,18 @@ config.json : code/config.ss
 
 # documentation
 docs/%.md : code/md.ss code/docs/%.ss
-	$(call exercise, "(put-md '$(@F:.md=))")
+	$(call exercise, "(put-doc '$(@F:.md=))")
 
-$(readme-splice) : docs/TESTS.md
-	cp $< $@
+$(readme-splice) : $(track-documentation)
+	cp docs/TESTS.md $@
 
 # exercises
-exercises/% : code/track.ss code/exercises/%/* code/stub-makefile
+exercises/% : code/track.ss code/md.ss code/exercises/%/* code/stub-makefile code/docs/tests.ss
```

Comment:
exercises are tied enough to documentation now that they should rebuild when `md.ss` changes